ra/registration-authority.go

```diff
@@ -87,6 +88,17 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(request core.Authorization
 		return authz, err
 	}
 
+	// Check CAA records for the requested identifier
+	present, valid, err := ra.VA.CheckCAARecords(identifier)
+	if err != nil {
+		return authz, err
+	}
+	ra.log.Audit(fmt.Sprintf("Checked CAA records for %s [Present: %v, Valid for issuance: %v]", identifier.Value, present, valid))
```

Comment:
Please put this mark right above the Audit call:
```
// AUDIT[ Certificate Requests ] 11917fa4-10ef-4e0d-9105-bacbe7836a3c
```

If you would, please also add authz.RegistrationID to it.